src/components/dashboard/DashboardPage.jsx

```diff
@@ -38,37 +43,61 @@ const DashboardPage = () => {
   }, [history, state]);
 
   const userFirstName = useMemo(() => authenticatedUser?.name.split(' ').shift(), [authenticatedUser]);
-  const CoursesTabComponent = (
-    <>
-      <Alert
-        variant="success"
-        show={isActivationAlertOpen}
-        onClose={closeActivationAlert}
-        className="mt-3"
-        dismissible
-      >
-        {LICENCE_ACTIVATION_MESSAGE}
-      </Alert>
-      <Row className="py-5">
-        <CourseEnrollmentsContextProvider>
-          <CourseEnrollmentFailedAlert className="mt-0 mb-3" enrollmentSource={ENROLLMENT_SOURCE.DASHBOARD} />
-          <MainContent>
-            <DashboardMainContent />
-          </MainContent>
-          <MediaQuery minWidth={breakpoints.large.minWidth}>
-            {matches => (matches ? (
-              <Sidebar data-testid="sidebar">
-                <DashboardSidebar />
-              </Sidebar>
-            ) : null)}
-          </MediaQuery>
-        </CourseEnrollmentsContextProvider>
-        <IntegrationWarningModal isOpen={enterpriseConfig.showIntegrationWarning} />
-        {subscriptionPlan && showExpirationNotifications && <SubscriptionExpirationModal />}
-      </Row>
-    </>
-  );
   const PAGE_TITLE = `Dashboard - ${enterpriseConfig.name}`;
+  const tabs = useMemo(() => {
+    const tabsList = [(
+      <Tab eventKey="courses" title="Courses">
+        <CoursesTabComponent canOnlyViewHighlightSets={canOnlyViewHighlightSets} />
+      </Tab>),
+    (
+      <Tab
+        eventKey="programs"
+        title="Programs"
+        disabled={learnerProgramsListData.length === 0}
+      >
+        {learnerProgramsListData > 0 && (
+          <ProgramListingPage
+            canOnlyViewHighlightSets={canOnlyViewHighlightSets}
+            programData={
+              {
+                data: learnerProgramsListData,
+                error: programsFetchError,
+              }
+            }
+          />
+        )}
+      </Tab>
+    )];
+    if (features.FEATURE_ENABLE_PATHWAY_PROGRESS) {
+      tabsList.push(
+        <Tab
+          eventKey="pathways"
+          title="Pathways"
+          disabled={pathwayProgressData.length === 0}
+        >
+          {pathwayProgressData.length > 0 && (
+            <PathwayProgressListingPage
+              canOnlyViewHighlightSets={canOnlyViewHighlightSets}
+              pathwayData={
+                {
+                  data: pathwayProgressData,
+                  error: pathwayFetchError,
+                }
+              }
```

Comment:
nit: I might recommend splitting this object into 2 distinct props, e.g., `pathwayProgressData` and `pathwayFetchError` as I believe it'll make reading these data easier to grok/parse within the `PathwayProgressListingPage` component itself.

Similar comment for `ProgramListingPage` above.